### DISCUSSION TITLE
Prediction interval vs. confidence interval

### DISCUSSION THREAD

**Discussion by MarleneBs** [awslabs/gluonts#2372] [Q&A] [answered]
I discussed with my supervisor that a confidence interval and a prediction interval are not the same thing. 
However, in the documentation for the plot function of the forecast object, the whole thing is used synonymously. In the description it says confidence interval and the parameters to be set are called forecast interval. Which of the two is now displayed in the plot function for the forecast object?

Translated with www.DeepL.com/Translator (free version)

**Comment by codingWhale13** [comment, marked as answer]
That's a good point. I'm not sure myself but maybe this helps a bit:
As you can see [here](https://github.com/awslabs/gluonts/blob/e89f0074faa8ff044cb8fbca0c9e5acde9d42b81/src/gluonts/model/forecast.py#L386), `plot` just uses the `quantile` method of `Forecast`. For example, for a 90% interval, the 0.05 and 0.95 quantiles are used for plotting.

**Reply by MarleneBs** [reply]
thank you, yes i know that, but i have found so little in the literature so far as to how both are defined in each case.

**Reply by MarleneBs** [reply]
https://stats.stackexchange.com/questions/16493/difference-between-confidence-intervals-and-prediction-intervals
Check this out, I am not sure but based on this i would say that the intervalls meant with the plot functions are the confidence intervals, what would you say ?

**Reply by MarleneBs** [reply]
But in this article it is kind of difficult explained, i am not sure if i get it correctly

**Reply by jgasthaus** [reply]
Models in GluonTS compute the predictive distribution $P(z_t|z_{1:t-1})$ for the next observation $z_t$ given the past observations $z_{1:t-1}$. What we are plotting are quantiles of this distribution, which can be seen as a _prediction interval_.

**Reply by MarleneBs** [reply]
Thank you very much

**Reply by MarleneBs** [reply]
I have to get back to you, 
I don't really understand it. In all literature there is a separation between prediction interval and confidence interval. Why is it used as a synonym in the documentation:
https://ts.gluon.ai/stable/_modules/gluonts/model/forecast.html#Forecast.plot

**Reply by MarleneBs** [reply]
I think i have found a definition that is equivalent to your statement 
thank you

**Reply by codingWhale13** [reply]
You're right, they are two different things. As @jgasthaus was saying, in the GluonTS context, we are concerned with the **prediction interval**.

I updated the documentation in #2373 (Might take a while to update on the [website](https://ts.gluon.ai/stable/_modules/gluonts/model/forecast.html#Forecast.plot) as well).

**Reply by MarleneBs** [reply]
Thank you very very much, this helps me a lot
